dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -233,12 +235,13 @@ super-csv/2.2.0//super-csv-2.2.0.jar
 threeten-extra/1.5.0//threeten-extra-1.5.0.jar
 transaction-api/1.1//transaction-api-1.1.jar
 univocity-parsers/2.9.0//univocity-parsers-2.9.0.jar
+velocity-engine-core/2.2//velocity-engine-core-2.2.jar
```

Comment:
We do not need `velocity-engine-core-2.2.jar`. We add exclusion before:
https://github.com/apache/spark/blob/b5101d20850b7c3ddc03cece8088a7b34b683084/pom.xml#L1230-L1233.

Reply:
I will update the exclusion to the new package `velocity-engine-core` replaced `velocity` in the `avro-ipc` dependencies